src/uci.cpp

```diff
@@ -290,8 +291,15 @@ void UCI::loop(int argc, char* argv[]) {
       else if (token == "d")        sync_cout << pos << sync_endl;
       else if (token == "eval")     sync_cout << Eval::trace(pos) << sync_endl;
       else if (token == "compiler") sync_cout << compiler_info() << sync_endl;
-      else if (token == "evalnn")   sync_cout << "NNUE evaluation: "
-                                    << Eval::NNUE::compute_eval(pos) << sync_endl;
+      else if (token == "evalnn") {
+        sync_cout << "NNUE evaluation: " << IO_UNLOCK;
```

Comment:
do we need IO_UNLOCK ?

Reply:
Yes, as sync_cout uses IO_LOCK but we don't have sync_endl which has IO_UNLOCK in this line and it just doesn't proceed further. Alternatively we can use std::cout here.

Reply:
Function` to_cp()` is not new, it existed already, I just moved it.

Reply:
OK, let me have a look.